internal/scorecard/plugins/labels_test.go

```diff
@@ -51,9 +52,15 @@ func TestBasicShortNames(t *testing.T) {
 				return
 			}
 
-			basicTests := NewBasicTestSuite(BasicTestConfig{})
-			basicTests.ApplySelector(selector)
-			testsSelected := len(basicTests.Tests)
+			tests := make([]schelpers.Test, 0)
+
+			conf := BasicTestConfig{}
+			tests = append(tests, NewCheckSpecTest(conf))
+			tests = append(tests, NewCheckStatusTest(conf))
+			tests = append(tests, NewWritingIntoCRsHasEffectTest(conf))
```

Comment:
This code would be idiomatic if each line were combined into one append:
```go
			tests = append(tests,
				NewCheckSpecTest(conf),
				NewCheckStatusTest(conf),
				NewWritingIntoCRsHasEffectTest(conf),
			)
```
Ditto below.

Reply:
I'll give that a try in the next refactor PR, good suggestion.